File: app/src/main/java/com/example/whattodo/CC_CardBackFragment.java
package com.example.whattodo;


import android.app.Fragment;
import android.os.Bundle;
import android.text.TextUtils;
import android.util.Log;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.example.whattodo.CCFragment.CCSecureCodeFragment;
import com.example.whattodo.CCUtils.FontTypeChange;

import butterknife.BindView;
import butterknife.ButterKnife;


/**
 * A simple {@link Fragment} subclass.
 */
public class CC_CardBackFragment extends Fragment {

    @BindView(R.id.tv_cvv)
    TextView tv_cvv;
    FontTypeChange fontTypeChange;

    CC_ItemCreditCardActivity activity;
    CCSecureCodeFragment securecode;

    public CC_CardBackFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment

        Log.d("BF", "onCreateView: ");
        View view=inflater.inflate(R.layout.fragment_card_back, container, false);
        ButterKnife.bind(this, view);

        tv_cvv = view.findViewById(R.id.tv_cvv);

        fontTypeChange=new FontTypeChange(getActivity());
        tv_cvv.setTypeface(fontTypeChange.get_fontface(1));

        activity = (CC_ItemCreditCardActivity) getActivity();
        securecode = activity.secureCodeFragment;
        securecode.setCvv(tv_cvv);

        if(!TextUtils.isEmpty(securecode.getValue()))
            tv_cvv.setText(securecode.getValue());

        return view;
    }

}

File: app/src/main/java/com/example/whattodo/CC_CardBackFragment.kt
package com.example.whattodo

import android.os.Bundle
import android.text.TextUtils
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import butterknife.BindView
import butterknife.ButterKnife
import com.example.whattodo.CCFragment.CCSecureCodeFragment
import com.example.whattodo.CCUtils.FontTypeChange

/**
 * A simple [Fragment] subclass.
 */
class CC_CardBackFragment : Fragment() {
    @BindView(R.id.tv_cvv)
    private lateinit var tv_cvv: TextView
    private lateinit var fontTypeChange: FontTypeChange
    private lateinit var cc_activity: CC_ItemCreditCardActivity
    private lateinit var securecode: CCSecureCodeFragment

    private lateinit var viewOfLayout: View

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        viewOfLayout = inflater.inflate(R.layout.fragment_card_back, container, false)
        ButterKnife.bind(this, viewOfLayout)

        tv_cvv = viewOfLayout.findViewById(R.id.tv_cvv)
        fontTypeChange = FontTypeChange(activity)
        tv_cvv.setTypeface(fontTypeChange!!.get_fontface(1))

        cc_activity = activity as CC_ItemCreditCardActivity
        securecode = cc_activity.secureCodeFragment
        securecode.setCvv(tv_cvv)

        if (!TextUtils.isEmpty(securecode.value)) tv_cvv.setText(securecode.value)

        return viewOfLayout
    }
}